addons/medical_damage/functions/fnc_parseConfigForInjuries.sqf

```diff
@@ -75,6 +77,7 @@ private _selectionSpecificDefault = getNumber (_damageTypesConfig >> "selectionS
     private _selectionSpecific = GET_NUMBER(_damageTypeSubClassConfig >> "selectionSpecific",_selectionSpecificDefault);
 
     GVAR(allDamageTypesData) setVariable [_className, [_thresholds, _selectionSpecific > 0, _woundTypes]];
+    GVAR(damageTypeCache) setVariable [_className, _className];
```

Comment:
```suggestion
    GVAR(damageTypeCache) setVariable [_className, _className];
    GVAR(damageTypeCache) setVariable ["#"+_className, _className];
```
Just in case anyone is using that somewhere